src/main/java/com/google/sps/data/SessionContext.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.User;
+import com.google.sps.data.NonPersistentUserRepository;
+import com.google.sps.data.UserRepository;
+import java.lang.Exception;
+
+public class SessionContext {
+
+    private UserService userService;
```

Comment:
nit: maybe use the final keyword here if userService won't be modified after the object is created

Reply:
Done.